google_compute_instance.network_interface.*.alias_ip_range assert error - max 1 array element permitted

### Terraform Version
+ Terraform version: 0.11.7  41e50bd32a8825a84535e353c3674af8ce799161
+ provider.google v1.16.2

### Affected Resource(s)
- google_compute_instance

### Terraform Configuration Files

resource "google_compute_instance" "instance1" {
    name                    = "instance1"
    machine_type            = "n1-standard-1"
    zone                    = "europe-west1-c"
    allow_stopping_for_update = true
        boot_disk {
        initialize_params {
            image           = "someimage"
            type            = "pd-standard"
            size            = "30"
        }
    }
      network_interface {
        subnetwork          = "net-0-sub-1" 
        subnetwork_project  = "someproject"
        address             = "172.27.1.11"
        alias_ip_range = [
            {
                "ip_cidr_range" = "172.27.1.12/32"
            },
            {
                "ip_cidr_range" = "172.27.1.13/32"
            }
        ]
    }

}

### Debug Output
018/08/10 13:57:48 [ERROR] root.module: eval: *terraform.EvalValidateResource, err: Warnings: []. Errors: [network_interface.0.alias_ip_range: attribute supports 1 item maximum, config has 2 declared]
2018/08/10 13:57:48 [ERROR] root.module: eval: *terraform.EvalSequence, err: Warnings: []. Errors: [network_interface.0.alias_ip_range: attribute supports 1 item maximum, config has 2 declared]



### Expected Behavior
There is a possibility to add couple of ip alias ranges from gcloud console 
Rest equivalent:
 "networkInterfaces": [
    {
      "aliasIpRanges": [
        {
          "ipCidrRange": "172.27.1.12/32"
        },
        {
          "ipCidrRange": "172.27.1.13/32"
        }
      ],
      "fingerprint": "00000=",
      "kind": "compute#networkInterface",
      "name": "nic0",
      "network": "somenetwork",
      "networkIP": "172.27.1.11",
      "subnetwork": "some subnetwork"
    }
  ],

Tfstate after importing existing resource with two ip alias ranges:
"network_interface.0.alias_ip_range.0.ip_cidr_range": "172.27.1.12/32",
"network_interface.0.alias_ip_range.0.subnetwork_range_name": "",
"network_interface.0.alias_ip_range.1.ip_cidr_range": "172.27.1.13/32",
"network_interface.0.alias_ip_range.1.subnetwork_range_name": "",

### Actual Behavior
Error: module.module.google_compute_instance.instance1: network_interface.0.alias_ip_range: attribute supports 1 item maximum, config has 2 declared

### Steps to Reproduce
1. Add following alias ip range block inside ntwork_interface block:
        alias_ip_range = [
            {
                "ip_cidr_range" = "172.27.1.12/32"
            },
            {
                "ip_cidr_range" = "172.27.1.13/32"
            }
        ]
2. Terraform apply

--- a/google/resource_compute_instance.go
+++ b/google/resource_compute_instance.go
@@ -218,7 +218,6 @@ func resourceComputeInstance() *schema.Resource {
 
 						"alias_ip_range": &schema.Schema{
 							Type:     schema.TypeList,
-							MaxItems: 1,
 							Optional: true,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{

--- a/google/resource_compute_instance_test.go
+++ b/google/resource_compute_instance_test.go
@@ -1576,7 +1576,7 @@ resource "google_compute_instance" "foobar" {
 	labels {
 		my_key       = "my_value"
 		my_other_key = "my_other_value"
-    }
+	}
 }
 `, instance)
 }
@@ -2788,83 +2788,83 @@ data "google_compute_image" "my_image" {
 }
 
 resource "google_compute_instance" "foobar" {
-  name = "%s"
-  machine_type = "n1-standard-1"
-  zone = "us-east1-d"
+	name = "%s"
+	machine_type = "n1-standard-1"
+	zone = "us-east1-d"
 
-  boot_disk {
-    initialize_params {
-      image = "${data.google_compute_image.my_image.self_link}"
-    }
-  }
+	boot_disk {
+		initialize_params {
+			image = "${data.google_compute_image.my_image.self_link}"
+		}
+	}
 
-  network_interface {
-    network = "default"
-  }
+	network_interface {
+		network = "default"
+	}
 
-  scheduling {
-    # Instances with guest accelerators do not support live migration.
-    on_host_maintenance = "TERMINATE"
-  }
+	scheduling {
+		# Instances with guest accelerators do not support live migration.
+		on_host_maintenance = "TERMINATE"
+	}
 
-  guest_accelerator {
-    count = %d
-    type = "nvidia-tesla-k80"
-  }
+	guest_accelerator {
+		count = %d
+		type = "nvidia-tesla-k80"
+	}
 }`, instance, count)
 }
 
 func testAccComputeInstance_minCpuPlatform(instance string) string {
 	return fmt.Sprintf(`
 data "google_compute_image" "my_image" {
-  family  = "debian-9"
-  project = "debian-cloud"
+	family  = "debian-9"
+	project = "debian-cloud"
 }
 
 resource "google_compute_instance" "foobar" {
-  name = "%s"
-  machine_type = "n1-standard-1"
-  zone = "us-east1-d"
+	name = "%s"
+	machine_type = "n1-standard-1"
+	zone = "us-east1-d"
 
-  boot_disk {
-    initialize_params {
-      image = "${data.google_compute_image.my_image.self_link}"
-    }
-  }
+	boot_disk {
+		initialize_params {
+			image = "${data.google_compute_image.my_image.self_link}"
+		}
+	}
 
-  network_interface {
-    network = "default"
-  }
+	network_interface {
+		network = "default"
+	}
 
-  min_cpu_platform = "Intel Haswell"
+	min_cpu_platform = "Intel Haswell"
 }`, instance)
 }
 
 func testAccComputeInstance_primaryAliasIpRange(instance string) string {
 	return fmt.Sprintf(`
 data "google_compute_image" "my_image" {
-  family  = "debian-9"
-  project = "debian-cloud"
+	family  = "debian-9"
+	project = "debian-cloud"
 }
 
 resource "google_compute_instance" "foobar" {
-  name = "%s"
-  machine_type = "n1-standard-1"
-  zone = "us-east1-d"
+	name = "%s"
+	machine_type = "n1-standard-1"
+	zone = "us-east1-d"
 
-  boot_disk {
-    initialize_params {
-      image = "${data.google_compute_image.my_image.self_link}"
-    }
-  }
+	boot_disk {
+		initialize_params {
+			image = "${data.google_compute_image.my_image.self_link}"
+		}
+	}
 
-  network_interface {
-    network = "default"
+	network_interface {
+		network = "default"
 
-    alias_ip_range {
-      ip_cidr_range = "/24"
-    }
-  }
+		alias_ip_range {
+			ip_cidr_range = "/24"
+		}
+	}
 }`, instance)
 }
 
@@ -2893,31 +2893,31 @@ resource "google_compute_subnetwork" "inst-test-subnetwork" {
 	}
 }
 resource "google_compute_instance" "foobar" {
-  name         = "%s"
-  machine_type = "n1-standard-1"
-  zone         = "us-east1-d"
+	name         = "%s"
+	machine_type = "n1-standard-1"
+	zone         = "us-east1-d"
 
-  boot_disk {
-    initialize_params {
-      image = "${data.google_compute_image.my_image.self_link}"
-    }
-  }
+	boot_disk {
+		initialize_params {
+			image = "${data.google_compute_image.my_image.self_link}"
+		}
+	}
 
-  network_interface {
-    subnetwork = "${google_compute_subnetwork.inst-test-subnetwork.self_link}"
+	network_interface {
+		subnetwork = "${google_compute_subnetwork.inst-test-subnetwork.self_link}"
 
-    alias_ip_range = [
-			{
-				subnetwork_range_name = "${google_compute_subnetwork.inst-test-subnetwork.secondary_ip_range.0.range_name}"
-				ip_cidr_range         = "172.16.0.0/24"
-			},
-			{
-				subnetwork_range_name = "${google_compute_subnetwork.inst-test-subnetwork.secondary_ip_range.1.range_name}"
-				ip_cidr_range         = "10.1.0.0/20"
-			}
-		]
+		alias_ip_range = [
+		{
+			subnetwork_range_name = "${google_compute_subnetwork.inst-test-subnetwork.secondary_ip_range.0.range_name}"
+			ip_cidr_range         = "172.16.0.0/24"
+		},
+		{
+			subnetwork_range_name = "${google_compute_subnetwork.inst-test-subnetwork.secondary_ip_range.1.range_name}"
+			ip_cidr_range         = "10.1.0.0/20"
+		}
+	]
 
-  }
+	}
 }`, network, subnet, instance)
 }
 
@@ -2946,23 +2946,22 @@ resource "google_compute_subnetwork" "inst-test-subnetwork" {
 	}
 }
 resource "google_compute_instance" "foobar" {
-  name         = "%s"
-  machine_type = "n1-standard-1"
-  zone         = "us-east1-d"
+	name         = "%s"
+	machine_type = "n1-standard-1"
+	zone         = "us-east1-d"
 
-  boot_disk {
-    initialize_params {
-      image = "${data.google_compute_image.my_image.self_link}"
-    }
-  }
+	boot_disk {
+		initialize_params {
+			image = "${data.google_compute_image.my_image.self_link}"
+		}
+	}
 
-  network_interface {
-    subnetwork = "${google_compute_subnetwork.inst-test-subnetwork.self_link}"
-
-    alias_ip_range {
-      ip_cidr_range = "10.0.1.0/24"
-    }
-  }
+	network_interface {
+		subnetwork = "${google_compute_subnetwork.inst-test-subnetwork.self_link}"
+		alias_ip_range {
+			ip_cidr_range = "10.0.1.0/24"
+		}
+	}
 }`, network, subnet, instance)
 }
 

--- a/google/resource_compute_instance_test.go
+++ b/google/resource_compute_instance_test.go
@@ -2887,6 +2887,10 @@ resource "google_compute_subnetwork" "inst-test-subnetwork" {
 		range_name = "inst-test-secondary"
 		ip_cidr_range = "172.16.0.0/20"
 	}
+	secondary_ip_range {
+		range_name = "inst-test-tertiary"
+		ip_cidr_range = "10.1.0.0/16"
+	}
 }
 resource "google_compute_instance" "foobar" {
   name         = "%s"
@@ -2902,10 +2906,17 @@ resource "google_compute_instance" "foobar" {
   network_interface {
     subnetwork = "${google_compute_subnetwork.inst-test-subnetwork.self_link}"
 
-    alias_ip_range {
-      subnetwork_range_name = "${google_compute_subnetwork.inst-test-subnetwork.secondary_ip_range.0.range_name}"
-      ip_cidr_range         = "172.16.0.0/24"
-    }
+    alias_ip_range = [
+			{
+				subnetwork_range_name = "${google_compute_subnetwork.inst-test-subnetwork.secondary_ip_range.0.range_name}"
+				ip_cidr_range         = "172.16.0.0/24"
+			},
+			{
+				subnetwork_range_name = "${google_compute_subnetwork.inst-test-subnetwork.secondary_ip_range.1.range_name}"
+				ip_cidr_range         = "10.1.0.0/20"
+			}
+		]
+
   }
 }`, network, subnet, instance)
 }
@@ -2928,6 +2939,10 @@ resource "google_compute_subnetwork" "inst-test-subnetwork" {
 	secondary_ip_range {
 		range_name    = "inst-test-secondary"
 		ip_cidr_range = "172.16.0.0/20"
+	}
+	secondary_ip_range {
+		range_name = "inst-test-tertiary"
+		ip_cidr_range = "10.1.0.0/16"
 	}
 }
 resource "google_compute_instance" "foobar" {

